### PR TITLE
LinkingAlps examples (from Andreas)

### PR DESCRIPTION
**Pull request by ue71603**
not yet done....

### CODE REVIEW

**Review by skinkie — DISMISSED** [review]

**Review by skinkie — APPROVED** [review]

**Review by sgrossberndt — APPROVED** [review]

**Comment by sgrossberndt** [comment]
This pull request cannot be opened again and is still shown as merged although it is not part of changes_for_v1.1 anymore: I removed it from the commit history by force push as the examples did not validate. It should have been rebased before merge. Sorry. I openend a new pull request at https://github.com/VDVde/OJP/pull/373

```
examples/linkingalps/LA_ARIA_TR_Response_with_error.xml:125: element TrackStart: Schemas validity error : Element '{http://www.vdv.de/ojp}TrackStart': This element is not expected. Expected is one of ( {http://www.vdv.de/ojp}TrackSectionStart, {http://www.vdv.de/ojp}TrackSectionEnd, {http://www.vdv.de/ojp}LinkProjection, {http://www.vdv.de/ojp}RoadName, {http://www.vdv.de/ojp}Duration, {http://www.vdv.de/ojp}Length, {http://www.vdv.de/ojp}Extension ).
examples/linkingalps/LA_ARIA_TR_Response_with_error.xml:202: element TrackStart: Schemas validity error : Element '{http://www.vdv.de/ojp}TrackStart': This element is not expected. Expected is one of ( {http://www.vdv.de/ojp}TrackSectionStart, {http://www.vdv.de/ojp}TrackSectionEnd, {http://www.vdv.de/ojp}LinkProjection, {http://www.vdv.de/ojp}RoadName, {http://www.vdv.de/ojp}Duration, {http://www.vdv.de/ojp}Length, {http://www.vdv.de/ojp}Extension ).
examples/linkingalps/LA_ARIA_TR_Response_with_error.xml:253: element TrackStart: Schemas validity error : Element '{http://www.vdv.de/ojp}TrackStart': This element is not expected. Expected is one of ( {http://www.vdv.de/ojp}TrackSectionStart, {http://www.vdv.de/ojp}TrackSectionEnd, {http://www.vdv.de/ojp}LinkProjection, {http://www.vdv.de/ojp}RoadName, {http://www.vdv.de/ojp}Duration, {http://www.vdv.de/ojp}Length, {http://www.vdv.de/ojp}Extension ).
examples/linkingalps/LA_ARIA_TR_Response_with_error.xml fails to validate
```

```
examples/linkingalps/LA_STA_TR_Response.xml:537: element TrackStart: Schemas validity error : Element '{http://www.vdv.de/ojp}TrackStart': This element is not expected. Expected is one of ( {http://www.vdv.de/ojp}TrackSectionStart, {http://www.vdv.de/ojp}TrackSectionEnd, {http://www.vdv.de/ojp}LinkProjection, {http://www.vdv.de/ojp}RoadName, {http://www.vdv.de/ojp}Duration, {http://www.vdv.de/ojp}Length, {http://www.vdv.de/ojp}Extension ).
examples/linkingalps/LA_STA_TR_Response.xml:563: element TrackStart: Schemas validity error : Element '{http://www.vdv.de/ojp}TrackStart': This element is not expected. Expected is one of ( {http://www.vdv.de/ojp}TrackSectionStart, {http://www.vdv.de/ojp}TrackSectionEnd, {http://www.vdv.de/ojp}LinkProjection, {http://www.vdv.de/ojp}RoadName, {http://www.vdv.de/ojp}Duration, {http://www.vdv.de/ojp}Length, {http://www.vdv.de/ojp}Extension ).
examples/linkingalps/LA_STA_TR_Response.xml:589: element TrackStart: Schemas validity error : Element '{http://www.vdv.de/ojp}TrackStart': This element is not expected. Expected is one of ( {http://www.vdv.de/ojp}TrackSectionStart, {http://www.vdv.de/ojp}TrackSectionEnd, {http://www.vdv.de/ojp}LinkProjection, {http://www.vdv.de/ojp}RoadName, {http://www.vdv.de/ojp}Duration, {http://www.vdv.de/ojp}Length, {http://www.vdv.de/ojp}Extension ).
examples/linkingalps/LA_STA_TR_Response.xml fails to validate
```

**Review by AndreasAtSBB — COMMENTED** [review]
New examples added, including error answer.

File: examples/linkingalps/LA_5T_TR_Request .xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+							<Text>Origin</Text>
+						</Name>
+					</PlaceRef>
+					<DepArrTime>2023-06-13T17:53:00.000Z</DepArrTime>
+				</Origin>
+				<Destination>
+					<PlaceRef>
+						<StopPlaceRef>gtt:1992</StopPlaceRef>
+						<Name>
+							<Text>Desitination</Text>
```
Comment (trurlurl) [inline examples/linkingalps/LA_5T_TR_Request .xml:22]: Destination

Comment (skinkie) [reply examples/linkingalps/LA_5T_TR_Request .xml:22]: Even worse: this file has a space in it.

Comment (sgrossberndt) [reply examples/linkingalps/LA_5T_TR_Request .xml:22]: I fixed that in https://github.com/VDVde/OJP/pull/374

Comment (trurlurl) [reply examples/linkingalps/LA_5T_TR_Request .xml:22]: I wanted to approve it, but it somehow didn't work...

**Comment by sgrossberndt** [comment]
The actual merged pull request which is part of changes_for_v1.1 is https://github.com/VDVde/OJP/pull/374
Sorry for the hassle.

**Pull request merged**